var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -130,7 +130,7 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("fftw-api@3")
     depends_on("libxc@3.0.0:")
     depends_on("libxc@4.0.0:", when="@7.2.0:")
-    depends_on("libxc@:6", when="@:7.6.1")
+    depends_on("libxc@:7", when="@:7.6.1")
```

Comment:
```suggestion
    depends_on("libxc@:7", when="@:7.5")
```
if the patch is applied from 7.6 onwards